torch/csrc/jit/runtime/register_prim_ops_fulljit.cpp

```diff
@@ -127,6 +128,48 @@ RegisterOperators reg(
            };
          },
          aliasAnalysisSpecialCase()),
+     Operator(
+         prim::AllocateStorage, /*size, device*/
+         [](const Node* node) -> Operation {
+           size_t total_size = node->i(attr::total_size);
+           auto device = static_cast<DeviceType>(node->i(attr::device));
+           return [total_size, device](Stack* stack) {
+             at::DataPtr buffer =
+                 MemoryPlanner::allocate_buffer(total_size, device);
+             auto storage = c10::Storage(
+                 c10::Storage::use_byte_size_t(),
+                 total_size,
+                 std::move(buffer),
+                 /*allocator=*/nullptr,
+                 /*resizable=*/false);
```

Comment:
what is the resizable false mean here ? could you add a comment ? i guess this is because we want to exactly model the size